org/enigma/EnigmaWriter.java

```diff
@@ -620,6 +620,14 @@ protected void populateShaders() {
 	}
 
 	protected void populateFonts() {
+		// no dpi scaling, since 96 is the Windows default
+		// it is the default for GM8, GMS1.4, and GMS2 and
+		// is independent of the current monitor or IDE settings
+		final int screenRes = 96;
```

Comment:
I'd have called this defaultDPI or something, but this is fine.

Reply:
If we add a setting later for the user to specify this per-project, then we'll bother to rename it. For now, this accomplishes what we want.